core/src/MsalIdHelper.cs

```diff
@@ -66,38 +66,32 @@ internal static class MsalIdParameter
     /// </summary>
     internal static class MsalIdHelper
```

Comment:
>MsalIdHelper [](start = 26, length = 12)

IMO, this thing is static and should be caching everything. Having to even redo the values here is odd and only needed for testing. What about optimizing it for run-time and use from reflection or internals visible to clean it up (e.g. via the ForceInitialize in the module initializer)...? This thing is right now waste of CPU cycles